modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -127,6 +131,19 @@ export default {
     moduleOptions() {
       return moduleOptionsBody(this.type);
     },
+    editorOptions() {
+      const defaultOptions = moduleOptionsBody(this.type).defaultOptions;
```

Comment:
Seems like you can use moduleOptions here rather than moduleOptionsBody, no?

Reply:
You are correct.